src/components/facet/facetVertical.js

```diff
@@ -453,6 +453,7 @@ FacetVertical.prototype._updateSparkline = function() {
 			selectedSparklinePath.appendTo(sparkline);
 
 			totalSparklinePath[0].classList.add('facet-sparkline-total');
+            totalSparklinePath[0].classList.add('facetkey-sparkline-total');
```

Comment:
is this a change specific to facet key?
if so, how would that affect other projects using this component?